### PR TITLE
feat(notes): label PR vs closed issues in changelog refs; blockquote PR-comment changelogs

### PR DESCRIPTION
**Pull request by goosewobbler**
Two changelog-rendering improvements (from design discussion on the standing-PR look).

## 1. Label PR vs closed issues; drop the inline rich card, keep the hovercard

`link`-mode refs now render **`(PR #503 · closes #500)`**:
- The PR link's visible text is `PR #503` (not a bare `#503`) on the canonical `/pull/503` URL. The non-bare text stops GitHub auto-expanding it into the inline rich-reference card (which duplicated the entry description); the canonical href keeps the hovercard.
- Closed issues render `closes [#500](…/issues/500), …` (omitted when none → `(PR #503)`).
- The PR is derived from the trailing `(#N)` of the squash-merge subject and stored as a new optional, additive `VersionChangelogEntry.prNumber` (it also stays in `issueIds`, so old manifests / unaware consumers render every ref via the fallback). **The PR is never guessed** — entries with no identifiable PR (non-squash, old manifest, non-GitHub repo) keep the prior `(#503, #499)` rendering. `escape`/`strip` modes and `@`-mention escaping are unchanged.

```
- Add thing (PR #503 · closes #500)
```

Wired through every producer (version commit parser → standing-PR + preview; notes git-log / conventional-changelog / version-output adapters → CHANGELOG.md) and all three render surfaces via the shared `renderIssueRefs` (which now returns the complete parens group; `markdown.ts` no longer double-wraps).

## 2. Blockquote the PR-comment changelogs

The standing-PR (per-row + combined footer) and preview-comment changelogs wrap their **content** in a Markdown blockquote for a left vertical bar that sets the changelog apart. The `<details>`/`<summary>` wrapper stays un-quoted so GitHub renders the disclosure reliably (a `> <details>` raw-HTML block renders unreliably); only the body is quoted. `CHANGELOG.md` (archival file) is left un-quoted.

```
<details><summary>Show all changes (2 changes, de-duplicated)</summary>

> **Added**
>
> - Add thing (PR #503 · closes #500)

</details>
```

## Verify on the live standing PR
Two things only GitHub's visual render can confirm (both degrade gracefully): that `PR [#503](…/pull/503)` keeps the hovercard while dropping the inline card, and that the blockquoted body nests cleanly inside the `<details>` (incl. the indented per-row pane). Since the standing PR dogfoods from `main`, the next update shows the real render.

## Tests
Parser PR-vs-issue extraction; `renderIssueRefs` (PR+closes / PR-only / fallback / escape / strip); blockquote wrapping on the standing-PR + preview surfaces. Full gate green (32/32); `schema:check` + root `biome check .` clean.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

<details open><summary><h3>Greptile Summary</h3></summary>

This PR adds two changelog-rendering improvements: (1) `renderIssueRefs` now returns the complete `(…)` group including parentheses and distinguishes the PR from closed issues with a `(PR #N · closes #M)` label when a `prNumber` is known; and (2) the body content inside `<details>` blocks on PR-comment surfaces (standing-PR and preview) is wrapped in a Markdown blockquote to visually set the changelog apart, while the `<details>`/`<summary>` HTML tags remain un-quoted to avoid GitHub rendering instability.

- **`prNumber` field** is added to `VersionChangelogEntry` and `ChangelogEntry` types, populated by the commit parsers (`commitParser.ts`, `git-log.ts`) when a squash-merge trailing `(#N)` is detected; `conventional-changelog.ts` deliberately omits it since its input is ambiguous.
- **`renderIssueRefs` API** now returns a self-contained parens group; all three call sites (`markdown.ts`, `changelog-region.ts`, `format.ts`) are updated to append the result bare rather than double-wrapping.
- **Blockquote helpers** (`blockquote()` in `format.ts`, inline in `wrapDetails()` in `changelog-region.ts`) prefix each non-blank inner line with `> ` and blank lines with `>` to keep one contiguous blockquote block.
</details>

<h3>Confidence Score: 5/5</h3>

Safe to merge — the changes are additive, backward-compatible, and thoroughly tested across all three render surfaces.

The `prNumber` field is optional and purely additive; old manifests and unaware consumers fall back to the existing flat-ref-list path. The `renderIssueRefs` API change (now returning the full parens group) is correctly propagated to every call site. Both blockquote helpers are straightforward string-prefix operations with no mutable state or async concerns. The 32-test gate is green and covers the key edge cases: PR-only, PR+closes, no-PR fallback, escape/strip modes, non-GitHub repos, and the defensive fold when `prNumber` is absent from `issueIds`.

No files require special attention. The parallel implementations of blockquote logic in `format.ts` and `changelog-region.ts` are minor duplication across package boundaries but present no functional risk.

<details><summary><h3>Important Files Changed</h3></summary>

| Filename | Overview |
|----------|----------|
| packages/core/src/changelogRefs.ts | Core API change: `renderIssueRefs` now returns a self-contained `(…)` group and accepts an optional `prNumber` to produce `(PR #N · closes #M)` output; defensive `allIds` fold ensures a `prNumber` absent from `issueIds` is never silently dropped. |
| packages/core/src/types.ts | Additive `prNumber?: string` field on `VersionChangelogEntry`; well-documented as optional/backward-compatible. |
| packages/version/src/changelog/commitParser.ts | Extracts trailing `(#N)` from the squash-merge subject as `prNumber`, prepends it to `bodyIssueIds` in `issueIds`, and strips it from the description; clean and well-tested. |
| packages/notes/src/input/git-log.ts | Mirrors `commitParser.ts` PR-extraction logic: issue IDs are already present (extracted message-wide), so only the subject strip and `prNumber` field assignment are added. |
| packages/notes/src/input/conventional-changelog.ts | Deliberately skips `prNumber` inference; a clear comment documents why — trailing `(#N)` in existing CHANGELOG text is ambiguous without a commit signal. |
| packages/release/src/preview/format.ts | Introduces `blockquote()` helper and applies it to the changelog body in `formatPackageChangelog` and the sync-bumped list; `<details>`/`<summary>` tags stay unquoted; caller sites updated to drop manual paren-wrapping of `renderIssueRefs`. |
| packages/release/src/standing-pr/changelog-region.ts | `wrapDetails` updated to inline the blockquote logic (`${indent}> ` / `${indent}>`) and receive `prNumber` through `entryLine`; disclosure tags stay indent-prefixed but unquoted. |
| packages/notes/src/output/markdown.ts | Correctly removes the manual `(${refs})` wrap now that `renderIssueRefs` returns the full parens group; passes `entry.prNumber` through. |
| packages/notes/src/input/version-output.ts | One-line additive change: forwards `prNumber` from the version-output entry through to the changelog input. |
| packages/core/test/unit/changelogRefs.spec.ts | Tests updated to expect the new self-contained parens group; new cases cover PR+closes, PR-only, fallback, escape, strip, and the defensive prNumber-absent-from-issueIds invariant. |
| packages/release/test/unit/changelog-region.spec.ts | Comprehensive new coverage for blockquote wrapping (footer and per-row pane) and PR-labeled refs in both single-package and multi-package scenarios. |

</details>

<details><summary><h3>Flowchart</h3></summary>

<a href="#gh-light-mode-only">

```mermaid
%%{init: {'theme': 'neutral'}}%%
flowchart TD
    A[Commit message] --> B{Squash-merge trailing N in subject?}
    B -- Yes --> C[Extract prNumber\nStrip N from subject text]
    B -- No --> D[prNumber = undefined]
    C --> E[issueIds = prNumber + body Closes refs]
    D --> F[issueIds = body refs only]
    E --> G[VersionChangelogEntry with prNumber and issueIds]
    F --> G

    G --> H[renderIssueRefs called with issueIds, mode, repoUrl, prNumber]
    H --> I{mode = strip?}
    I -- Yes --> J[Returns empty string]
    I -- No --> K{GitHub repoUrl AND prNumber set?}
    K -- Yes --> L[Returns PR-labeled group e.g. PR #503 closes #500]
    K -- No --> M{GitHub repoUrl resolvable?}
    M -- Yes --> N[Returns plain link group]
    M -- No or escape mode --> O[Returns escaped group]

    L --> P[Caller appends result bare after space]
    N --> P
    O --> P

    P --> Q{Output surface?}
    Q -- CHANGELOG.md --> R[Plain output no blockquote]
    Q -- PR comment surface --> S[Content wrapped in blockquote inside unquoted HTML details block]
```

</a>
<a href="#gh-dark-mode-only">

```mermaid
%%{init: {'theme': 'base', 'themeVariables': {"darkMode": true, "background": "#0d1117", "primaryColor": "#21262d", "primaryTextColor": "#e6edf3", "primaryBorderColor": "#8b949e", "lineColor": "#8b949e", "textColor": "#e6edf3", "edgeLabelBackground": "#161b22", "actorBkg": "#21262d", "actorBorder": "#8b949e", "actorTextColor": "#e6edf3", "actorLineColor": "#8b949e", "signalColor": "#8b949e", "signalTextColor": "#e6edf3", "noteBkgColor": "#373320", "noteBorderColor": "#d4a72c", "noteTextColor": "#f0e6c0", "labelBoxBkgColor": "#21262d", "labelBoxBorderColor": "#8b949e", "labelTextColor": "#e6edf3", "loopTextColor": "#e6edf3", "activationBkgColor": "#30363d", "activationBorderColor": "#8b949e"}}}%%
flowchart TD
    A[Commit message] --> B{Squash-merge trailing N in subject?}
    B -- Yes --> C[Extract prNumber\nStrip N from subject text]
    B -- No --> D[prNumber = undefined]
    C --> E[issueIds = prNumber + body Closes refs]
    D --> F[issueIds = body refs only]
    E --> G[VersionChangelogEntry with prNumber and issueIds]
    F --> G

    G --> H[renderIssueRefs called with issueIds, mode, repoUrl, prNumber]
    H --> I{mode = strip?}
    I -- Yes --> J[Returns empty string]
    I -- No --> K{GitHub repoUrl AND prNumber set?}
    K -- Yes --> L[Returns PR-labeled group e.g. PR #503 closes #500]
    K -- No --> M{GitHub repoUrl resolvable?}
    M -- Yes --> N[Returns plain link group]
    M -- No or escape mode --> O[Returns escaped group]

    L --> P[Caller appends result bare after space]
    N --> P
    O --> P

    P --> Q{Output surface?}
    Q -- CHANGELOG.md --> R[Plain output no blockquote]
    Q -- PR comment surface --> S[Content wrapped in blockquote inside unquoted HTML details block]
```

</a>
</details>

<sub>Reviews (2): Last reviewed commit: ["fix(notes): don&#39;t infer PR from ambiguou..."](https://github.com/goosewobbler/releasekit/commit/ad9557b12ff664ee33e717fc809eeb15f0143243) | [Re-trigger Greptile](https://app.greptile.com/api/retrigger?id=40855047)</sub>

### CODE REVIEW

File: packages/notes/src/input/conventional-changelog.ts [hunk 1]
```diff
@@ -83,6 +83,11 @@ function parseEntry(line: string, sectionType: ChangelogEntry['type']): Changelo
     rawText = scopeMatch[2] ?? rawText;
   }
 
+  // Squash-merge convention: a lone trailing `(#N)` is the PR GitHub appended. Mark it so the renderer
+  // can label it; the number still lands in issueIds (the full flat list) via extractIssueIds.
+  const prMatch = rawText.match(/\(#(\d+)\)\s*$/);
+  const prNumber = prMatch ? `#${prMatch[1]}` : undefined;
```
[the remark this answers is not in the document]
Comment (goosewobbler) [reply packages/notes/src/input/conventional-changelog.ts]: Good call — fixed in ad9557b1 by *removing* PR inference from this adapter rather than just commenting it. This path parses existing/external CHANGELOG.md text where a trailing `(#N)` is genuinely ambiguous and there's no commit signal to confirm, so it no longer sets `prNumber` (the ref still lands in `issueIds`). PR detection stays only in the commit-based parsers (`commitParser.ts`, `git-log.ts`), where the subject's trailing `(#N)` is the reliable squash-merge signal. Updated the test to assert the no-label behaviour.

File: packages/core/src/changelogRefs.ts [hunk 2]
```diff
@@ -33,33 +33,58 @@ export function parseGitHubOwnerRepo(repoUrl: string): { owner: string; repo: st
...
-        return `[#${num}](https://github.com/${ownerRepo.owner}/${ownerRepo.repo}/issues/${num})`;
-      }
       // escape mode, or link mode with no resolvable GitHub repo → literal `#NNN`.
-      return `\\#${num}`;
+      return ownerRepo
+        ? `[#${num}](https://github.com/${ownerRepo.owner}/${ownerRepo.repo}/issues/${num})`
+        : `\\#${num}`;
     })
     .join(', ');
+  return rendered ? `(${rendered})` : '';
```
[the remark this answers is not in the document]
Comment (goosewobbler) [reply packages/core/src/changelogRefs.ts:92]: Fixed in ad9557b1. `renderIssueRefs` now folds `prNumber` into a single `allIds` working list up front (deduped by numeric value), used by the empty-guard, the closes computation, and the fallback render — so a `prNumber` absent from `issueIds` can never be silently dropped in escape / non-GitHub mode. Added a defensive test (`renderIssueRefs([], 'escape', repo, '#503')` → `(\#503)`).
